src/azure-cli/setup.py

```diff
@@ -64,7 +64,7 @@
     'azure-loganalytics~=0.1.0',
     'azure-mgmt-advisor==9.0.0',
     'azure-mgmt-apimanagement~=0.2.0',
-    'azure-mgmt-appconfiguration~=1.0.1',
+    'azure-mgmt-appconfiguration~=2.0.0',
```

Comment:
Any breaking changes that might be introduced in the 2.0.0 release?

Reply:
Also, you'll have to add this new package to the `requirements.xxx` files too

Reply:
No breaking change in this preview version. From CLI's perspective, PR just introduces some new properties.